modules/tasks/tasks/src/main/java/org/motechproject/tasks/service/util/DataSourceObject.java

```diff
@@ -36,6 +38,10 @@ public boolean isFailIfNotFound() {
         return failIfNotFound;
     }
 
+    public boolean isNullWarningPublished() { return nullWarningPublished; }
```

Comment:
Why is this condensed to one line? The statement should start in a new line, similar to the closing brace.